new/language/tree/node.go

```diff
@@ -67,6 +67,45 @@ func (node *Node) Child(i int) *Node {
 	return node.tree.wrap(node.sitterNode.Child(i))
 }
 
+func (node *Node) EachPart(onText func(text string) error, onChild func(child *Node) error) error {
```

Comment:
This might be more descriptive:
```suggestion
func (node *Node) EachContentPart(onText func(text string) error, onChild func(child *Node) error) error {
```